DuckDuckGo/AuthConfirmationPromptView.swift

```diff
@@ -40,7 +40,7 @@ struct AuthConfirmationPromptView: View {
                 Button {
                     viewModel.cancelButtonPressed()
                 } label: {
-                    Text(UserText.autofillCancel)
```

Comment:
It turns out this string is in english in all languages, so updated to use `UserText.actionCancel` which is correctly translated